package.json

```diff
@@ -44,7 +45,7 @@
     "axios-mock-adapter": "^1.18.1",
     "core-js": "^3.6.5",
     "cross-env": "^7.0.2",
-    "cssnano": "^7.0.6",
+    "cssnano": "^4.1.10",
```

Comment:
Why are we downgrading ?

Reply:
I didnt notice it, thanks. I will reverted it.

Reply:
done